Combine 'Index mgmt' with 'Index mgmt in Kibana' (#2010)
elastic/docs-content: manage-data/data-store/index-basics.md

## Index management

> Elastic's **Index Management** features are an easy, convenient way to manage your cluster’s indices, [data streams](/manage-data/lifecycle/data-stream.md), [index templates](/manage-data/data-store/templates.md), and [enrich policies](/manage-data/ingest/transform-enrich/data-enrichment.md). Practicing good index management ensures your data is stored correctly and in the most cost-effective way possible.

Reviewer: ```suggestion
Elastic's **Index Management** features are an easy, convenient way to manage your cluster’s indices, [data streams](/manage-data/lifecycle/data-stream.md), [templates](/manage-data/data-store/templates.md), and [enrich policies](/manage-data/ingest/transform-enrich/data-enrichment.md). Practicing good index management ensures your data is stored correctly and in the most cost-effective way possible.
```

IMO for that link we should use the generic word `templates`, which has been introduced I think with the new IA. That doc (templates) explain that there are 2 types of templates: index templates and component templates, and that doc doesn't have good anchors to link specifically index-templates and component-templates (something to improve in a separate issue probably).

As the UI allows to configure both, I think "templates" here is a better choice.

#### Try it: Create an index template [_try_it_create_an_index_template]

In this tutorial, you’ll create an index template and use it to configure two new indices.

##### Step 1. Add a name and index pattern

1. In the **Index Templates** view, open the **Create template** wizard.

    :::{image} /manage-data/images/elasticsearch-reference-management_index_create_wizard.png
    :alt: Create wizard
    :screenshot:
    :::

2. In the **Name** field, enter `my-index-template`.
3. Set **Index pattern** to `my-index-*` so the template matches any index with that index pattern.
4. Leave **Data Stream**, **Priority**, **Version**, and **_meta field** blank or as-is.

##### Step 2. Add settings, mappings, and aliases

1. Add [component templates](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-cluster-put-component-template) to your index template.

    Component templates are pre-configured sets of mappings, index settings, and aliases you can reuse across multiple index templates. Badges indicate whether a component template contains mappings (**M**), index settings (**S**), aliases (**A**), or a combination of the three.

>     Component templates are optional. For this tutorial, do not add any component templates.

Reviewer: I still don't like much the link to the `PUT component template` API doc in a step that is about adding existing component templates to the index, but it's not a big deal, and the referenced API doc explains component templates properly also.

    :::{image} /manage-data/images/elasticsearch-reference-management_index_component_template.png
    :alt: Component templates page
    :screenshot:
    :::

> 2. Define index settings. These are optional. For this tutorial, leave this section blank.

Reviewer: ```suggestion
2. Define index settings directly in the index template. When used in conjunction with component templates, settings defined directly in the index template override any conflicting settings from the associated component templates.

    This step is optional. For this tutorial, leave this section blank.
```

> 3. Define a mapping that contains an [object](elasticsearch://reference/elasticsearch/mapping-reference/object.md) field named `geo` with a child [`geo_point`](elasticsearch://reference/elasticsearch/mapping-reference/geo-point.md) field named `coordinates`:

Reviewer: ```suggestion
3. Define mappings directly in the index template. When used in conjunction with component templates, these mappings override any conflicting definitions from the associated component templates.

    Define a mapping that contains an [object](elasticsearch://reference/elasticsearch/mapping-reference/object.md) field named `geo` with a child [`geo_point`](elasticsearch://reference/elasticsearch/mapping-reference/geo-point.md) field named `coordinates`:
```
